docs/source/theory/kfactors.rst

```diff
@@ -4,8 +4,16 @@ K-Factors
 Another useful tool that `pineko` includes is ``pineko kfactor`` which allows the embedding of a kfactor
 as a proper order in a grid. The usage is the following::
 
-  pineko kfactor GRIDS_FOLDER KFACTOR_FOLDER YAMLDB_PATH TARGET_FOLDER MAX_AS ORDER_EXISTS
+  pineko kfactor GRIDS_FOLDER KFACTOR_FOLDER YAMLDB_FILE TARGET_FOLDER PTO_TO_UPDATE [--order_exists]
 
 where ``GRIDS_FOLDER`` is the folder containing the grids to update, ``KFACTOR_FOLDER`` is the folder
-containing the kfactor files and ``YAMLDB_PATH`` is the path to the yamldb file of the requested dataset.
+containing the kfactor files and ``YAMLDB_FILE`` is the path to the yamldb file of the requested dataset.
 The other inputs have already been described in the previous section.
```

Comment:
I don't think we use `TARGET_FOLDER` any where else, do we? and even if we do - let's repeat it here for the sake of having a single place

Reply:
done in [3163be4](https://github.com/NNPDF/pineko/pull/161/commits/3163be4955b25bc48118002614d3f799284fa3f1)